gateway/grpc/README.md

```diff
@@ -0,0 +1,158 @@
+# gRPC on Gateway Controller
+
+End-to-end example of gRPC Load Balancing with gRPC:
+
+* Deploy gRPC application on GKE
+* Enable Gateways to handle both internet facing and internal-only traffic.
+* Verify gRPC LoadBalancing through Gateway
+
+---
+
+First deploy a GKE cluster with NEG enabled:
+
+```bash
+gcloud container  clusters create cluster-1 --machine-type "n1-standard-2" \
+ --zone us-central1-a  --num-nodes 2 --enable-ip-alias  \
+ --cluster-version "1.20"  -q
+```
+
+
+Create SSL Certificate
+
+Note, as of `5/6/21`, Gateway does not support [spec.listeners.tls.certificateRefs](https://cloud.google.com/kubernetes-engine/docs/how-to/gatewayclass-capabilities#gateway) so we have to upload a static cert to GCP.  Note we are creating two sets of certificates scoped for global and regional to account for external LB (global) and ILB (regional)
+
+```bash
+gcloud beta compute ssl-certificates create gcp-cert-grpc-global \
```

Comment:
Does it have to be created with the beta command? It's GA no?

Reply:
you're right, its GA now, changedupdating it now